### PR TITLE
Release notes for 10.1.6

### PR DESCRIPTION
**Pull request by jrudolph**

### CODE REVIEW

File: docs/src/main/paradox/release-notes/10.1.x.md [hunk 1]
```diff
@@ -2,18 +2,72 @@
 
 ## 10.1.6
 
-10.1.6 is the seventh release in the 10.1.x series of Akka HTTP.
+10.1.6 is the seventh release in the 10.1.x series of Akka HTTP. See the list below for the list of changes.
```
Comment (johanandren) [inline docs/src/main/paradox/release-notes/10.1.x.md]: see the list for the list

Comment (jrudolph) [reply docs/src/main/paradox/release-notes/10.1.x.md]: Thanks, fixed.

File: docs/src/main/paradox/release-notes/10.1.x.md [hunk 2]
```diff
@@ -2,18 +2,72 @@
...
 
 For a full overview you can also see the [10.1.6 milestone](https://github.com/akka/akka-http/milestone/44?closed=1):
 
-#### Fixes in akka-http-core
+#### Improvements
+
+##### akka-http-core
+
+ * Move RoutingSettings to akka-http module ([#2307](https://github.com/akka/akka-http/issues/2307))
+ * Use Collision-resistant maps in header parsing ([#2276](https://github.com/akka/akka-http/issues/2276))
```
Comment (ennru) [inline docs/src/main/paradox/release-notes/10.1.x.md]:
```suggestion
  * Use Collision-resistent maps in header parsing ([#2276](https://github.com/akka/akka-http/issues/2276))
```

File: docs/src/main/paradox/release-notes/10.1.x.md [hunk 3]
```diff
@@ -2,18 +2,72 @@
...
+ * Log time spent in a pool slot for simpler spotting of slow transition periods ([#2226](https://github.com/akka/akka-http/issues/2226))
+ * Replace existing HTTP Status Code 425 with TOO_EARLY ([#2255](https://github.com/akka/akka-http/issues/2255))
+ * Java API - call scala HttpEntity.toStrict that respects maxBytes ([#2287](https://github.com/akka/akka-http/issues/2287))
+ * Override `getForKeyCaseInsensitive` in `HttpMethods` ([#2303](https://github.com/akka/akka-http/issues/2303))
+ * Fix several warnings across all the modules ([#2292](https://github.com/akka/akka-http/issues/2292))
+ * Deprecate unofficial Microsoft media-types and add the official ones ([#2264](https://github.com/akka/akka-http/issues/2264))
+
+##### akka-http
+
+ * Use collision-resistant maps for formFieldMaps ([#2274](https://github.com/akka/akka-http/issues/2274))
```
Comment (ennru) [inline docs/src/main/paradox/release-notes/10.1.x.md]:
```suggestion
  * Use collision-resistent maps for formFieldMaps ([#2274](https://github.com/akka/akka-http/issues/2274))
```

File: docs/src/main/paradox/release-notes/10.1.x.md (hunk 2 above)
Comment (ennru) [reply docs/src/main/paradox/release-notes/10.1.x.md]: You're right, I checked with a less credible source.

**Pull request merged**